generators/client/templates/react/src/test/javascript/e2e/modules/administration/administration.spec.ts.ejs

```diff
@@ -0,0 +1,87 @@
+import { element, by, browser, $ } from 'protractor';
+import { waitUntilDisplayed, waitUntilHidden } from '../../util/utils';
+import NavBarPage from '../../page-objects/navbar-page';
+
+const expect = chai.expect;
+
+describe('Administration', () => {
+  let navBarPage: NavBarPage;
+
+  before(() => {
+    navBarPage = new NavBarPage(true);
+    navBarPage.autoSignIn();
+  });
+
+  describe('User management', () => {
+    it('should load user management', async () => {
+      navBarPage.clickOnAdminMenuItem('user-management');
+      expect(
+        await element
+          .all(by.css('h2'))
```

Comment:
Feel free to add unique ids to elements used in protractor tests as they are more reliable